docs/app/Examples/modules/Popup/Types/PopupHtmlExample.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Popup, Card, Rating, Image } from 'stardust'
+
+const users = [
+  {
+    name: 'Elliot Fu',
+    bio: 'Elliot has been a member since July 2012',
+    avatar: 'http://semantic-ui.com/images/avatar/small/elliot.jpg',
+  },
+  {
+    name: 'Stevie Feliciano',
+    bio: 'Stevie has been a member since August 2013',
+    avatar: 'http://semantic-ui.com/images/avatar/small/stevie.jpg',
+  },
+  {
+    name: 'Matt',
+    bio: 'Matt has been a member since July 2014',
+    avatar: 'http://semantic-ui.com/images/avatar/small/matt.jpg',
+  },
+]
+
+export default function () {
+  const IndividualCard = (
+    <Card>
+      <Image src='http://semantic-ui.com/images/movies/totoro-horizontal.jpg' />
+      <Card.Content>
+        <Card.Header>
+          My Neighbor Totoro
+        </Card.Header>
+        <Card.Description>
+          Two sisters move to the country with their father in order to be
+          closer to their hospitalized mother, and discover the surrounding
+          trees are inhabited by magical spirits.
+        </Card.Description>
+      </Card.Content>
+    </Card>
+  )
+
+  const content = <div>
+    <div className='header'>User Rating</div>
+    <div className='content'>
+      <Rating icon='star' defaultRating={3} maxRating={4} />
+    </div>
+  </div>
+
+  return <Popup
+    trigger={IndividualCard}
+    content={content}
+    hover
+  />
```

Comment:
Hm, I get this in my editor:

![image](https://cloud.githubusercontent.com/assets/5067638/19010287/a86760d4-8732-11e6-816a-d55689c9f632.png)

And this from the CLI:

![image](https://cloud.githubusercontent.com/assets/5067638/19010310/08cc59e8-8733-11e6-852c-d68c910e0459.png)

Maybe double check the linter setup and that the latest modules are installed?  I know NPM has had some issues as of late, sometimes requiring multiple installs to get all the modules you should've gotten the first go.  LMK if you have further issues and we can investigate deeper.